src/storage/prisma.ts

```diff
@@ -66,49 +67,15 @@ function storageToOperation(
     return operation;
 }
 
-export type PrismaStorageOptions = {
-    transactionRetryBackoff?: IBackOffOptions;
-};
-
-function getRetryTransactionsClient<T extends PrismaClient>(
```

Comment:
This retry is no longer needed since we know control the retries at a lower level, when the optimistic write fails.
This was needed for when the serializable transaction failed and it's the error that is being reported on Sentry.